spec/models/customer_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'rails_helper'
+
+describe Customer do
+  it "should incliude Her::Model" do
+    Customer.include?(Her::Model).should be true
```

Comment:
RSpec has a new expectation syntax: http://rspec.info/blog/2012/06/rspecs-new-expectation-syntax/

I would suggest to write

```
expect(Customer.include?(Her::Model)).to be_truthy
```